lib/parse/index.js

```diff
@@ -0,0 +1,22 @@
+/*!
+ * parse/index.js - JOSE Parser Entry Point
+ *
+ * Copyright (c) 2015 Cisco Systems, Inc. See LICENSE file.
+ */
+"use strict";
+
+var compact = require("./compact"),
+    json = require("./json");
+
+var parse = module.exports = function(input) {
+  if ("string" === typeof input) {
```

Comment:
> On Oct 2, 2015, at 12:15, Ian Remmel <notifications@github.com> wrote:
> 
> In lib/parse/index.js:
> 
> > @@ -0,0 +1,22 @@
> > +/*!
> > + * parse/index.js - JOSE Parser Entry Point
> > + *
> > + * Copyright (c) 2015 Cisco Systems, Inc. See LICENSE file.
> > + */
> > +"use strict";
> > +
> > +var compact = require("./compact"),
> > +    json = require("./json");
> > +
> > +var parse = module.exports = function(input) {
> > +  if ("string" === typeof input) {
> 
> All that said, are Buffers used explicitly elsewhere in the Jose family? I've had the Buffer shim have weird side effects in browsers, so if we're not already doing something that guarantees it's in the bundle, maybe it's best to leave the extra convenience out.
> 

Buffers are used pretty liberally, especially for all underlying the cipher operations.  Making sure it's in the bundle sounds like something we need to do, but it also sounds like it's a separate issue from this PR.


--
- m&m

Matthew A. Miller
< http://goo.gl/LM55L >